### ISSUE TITLE
Citation metadata file

### ISSUE THREAD

**Issue by leonlan** [INFORMSJoC/INFORMSJoC.github.io#4]
Hi!

The citation data presented on the repository does not match the actual citation data on Zenodo. On the repository, it is presented as:

```
@article{Naderi2023,
  author =        {Bahman Naderi, Rubén Ruiz, Vahid Roshanaei},
  publisher =     {INFORMS Journal on Computing},
  title =         {Repository for Mixed-Integer Programming Versus Constraint Programming for Shop Scheduling Problems: New Results and Outlook.},
  year =          {2023},
  doi =           {10.5281/zenodo.7541223},
  url =           {https://github.com/INFORMSJoC/2021.0326},
}  
```

But the Zenodo repository presents the metadata as follows:
![image](https://github.com/user-attachments/assets/5a24d7a4-a344-4e18-8e79-47b0fb5e8ce1)

See [zenodo.org/records/7541223](https://zenodo.org/records/7541223).

Another point that I would like to suggest is to include a CITATION metadata file in the repository. See [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) for details. Plugins like Zotero Connector can then be conveniently used to cite the repository data.

I'm not sure whether I should address this in the specific repository.

**Comment by tkralphs** [comment]
Thanks for the push on this. I'm aware of the CITATION metadata file and indeed, it would be very nice if every repo had one. I had on my radar to try to make that happen, but making it so is a bit tricky with the workflow that we have. The repo you are referencing has a DOI from Zenodo, which we were using when we first stated this Github initiative, but we have now moved to a system where we get a DOI directly from INFORMS Pubs. Some of the ways that we do all of this are in flux due to initiatives underway at INFORMS Pubs that will affect multiple journals. So stay tuned, I'll make sure this is part of the conversation as we move forward. Thanks!
